### PR TITLE
Inbox docs update

### PR DESCRIPTION
**Pull request by scopsy**
Updated page structure and content for inbox component: https://novu-inbox-docs-update.mintlify.app/inbox/react/get-started

### CODE REVIEW

File: inbox/react/advanced-configuration.mdx [hunk 1]
```diff
@@ -4,24 +4,6 @@ sidebarTitle: 'Advanced Configuration'
 description: 'Learn how to configure the Inbox component with advanced options'
 ---
 
-## Use your own backend and socket URL
```
Comment (scopsy) [inline inbox/react/advanced-configuration.mdx:7]: Moved to the bottom of the page

File: inbox/react/components/bell.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+title: '<Bell />'
```
Comment (scopsy) [inline inbox/react/components/bell.mdx:2]: Broke each component to it's own page

File: inbox/react/components/bell.mdx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      />
+    </Inbox>
+  );
+}
+```
+
+
+### Bring your own Popover
+
+component allows you to bring your own popover component along with custom components.
```
Comment (SokratisVidros) [inline inbox/react/components/bell.mdx:50]:
```suggestion
<Inbox/> can be mounted in your own popover component. For further customization, you can also use the `renderBell` and r`enderNotification` render props.
```

File: inbox/react/components/bell.mdx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+Below is an example of how you can bring your own popover component with Radix UI.
```
Comment (SokratisVidros) [inline inbox/react/components/bell.mdx:51]:
```suggestion
Below is an example of how to use `<Inbox/>` with [Radix UI](https://www.radix-ui.com/).
```

File: inbox/react/components/inbox.mdx [hunk 5]
```diff
@@ -0,0 +1,138 @@
+---
+title: '<Inbox />'
+---
+
+
+It renders an opinionated `<Inbox />` with a bell button, that triggers a popover on lock. The popover contains the notifications list and the user preferences.
```
Comment (SokratisVidros) [inline inbox/react/components/inbox.mdx:6]:
```suggestion
Renders `<Inbox />` with a bell button, that opens a popover on click. The popover contains the notifications list and the user preferences.
```

File: inbox/react/get-started.mdx [hunk 6]
```diff
@@ -1,42 +1,59 @@
...
-
-3. Run the app
-
-You can now run the app and [trigger](/concepts/trigger) the events to see the Inbox in action.
-
-<Frame caption="The Inbox Component">
-  <img src="/images/inbox/component.png" />
-</Frame>
+<Steps>
+    <Step title="Install the Inbox package">
```
Comment (SokratisVidros) [inline inbox/react/get-started.mdx:11]: Please fix indentation, here we use tabs or 4 spaces.

File: inbox/react/styling.mdx [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+
+## Customization Hierarchy
+
+The Inbox component is built to allow for multiple layers of styling, which allows the specificity required to style the Inbox to meet the requirements of your use case.
+
+Depending on the level of customization you need, you can choose to style the inbox using one of the following approaches:
+- [Appearance Prop](#appearance-prop)
+  - [Variables](#variables) - Global primitives such as buttons, popovers, dropdowns and etc...
+  - [Elements](#elements) - Style individual elements
+- [Render Notification](#render-notification-component) - Render a custom notification item with complete control
```
Comment (SokratisVidros) [inline inbox/react/styling.mdx:15]:
```suggestion
- [Custom notification rendering](#render-notification-component) - Render a custom notification item with complete control
```

**Comment by LetItRock** [comment]
https://novu-inbox-docs-update.mintlify.app/inbox/react/multiple-tabs
the image at the bottom of this page doesn't render

**Pull request merged**